spec/factories/defendants.rb

```diff
@@ -14,7 +14,6 @@
     end
     prosecution_case
     masterDefendantId { SecureRandom.uuid }
-    courtProceedingsInitiated { "2019-10-17 14:06:13" }
```

Comment:
Need to add:
proceedingsConcluded
isYouth

Reply:
To keeps things more simple, I have not added all non-required fields. They can be added in the future if need be.